rust/src/web/graphql/feetransfers/mod.rs

```diff
@@ -0,0 +1,237 @@
+use super::blocks::{Block, BlockWithCanonicity};
+use crate::{
+    block::{precomputed::PrecomputedBlock, store::BlockStore, BlockHash},
+    canonicity::{store::CanonicityStore, Canonicity},
+    command::{internal::InternalCommandWithData, store::CommandStore},
+    constants::MAINNET_GENESIS_HASH,
+    store::IndexerStore,
+};
+use async_graphql::{Context, Enum, InputObject, Object, Result, SimpleObject};
+use chrono::{DateTime, SecondsFormat};
+use std::sync::Arc;
+
+#[derive(SimpleObject)]
+pub struct Feetransfer {
+    pub state_hash: String,
+    pub fee: u64,
+    pub recipient: String,
+    #[graphql(name = "type")]
+    pub feetransfer_kind: String,
+    pub block_height: u32,
+    pub date_time: String,
+}
+
+pub struct FeetransferWithMeta {
+    /// Value canonicity
+    pub canonical: bool,
+    /// Value optional block
+    pub block: Option<PrecomputedBlock>,
+    /// Value feetranser
+    pub feetransfer: Feetransfer,
+}
+
+#[Object]
```

Comment:
Could you use `#[derive(SimpleObject)]` and avoid needing the functions?

Reply:
Unfortunately I can't if I want to refer to another resolver which I do in one of the filter branches